### PR TITLE
🐍 Add query node mock server

### LINKED ISSUE #156: Add Hydra query node mock server
Add hydra mock server to

- get data from the server
- use the mock server in UI tests

### PR DESCRIPTION
**Pull request by jodator**
Closes #156 

The mock server setup includes:

- Miragejs to mock requests responses
- ApolloClient to access query node graphql API
- Initial members queries and mocks – rest will be added when working on particular features

There's some amount of generated code but it looks like it's better than writing them by hand anyway.

### CODE REVIEW

File: packages/ui/webpack.config.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```
Comment (krzysztof-jelski) [inline packages/ui/webpack.config.js:1]: This looks strange here. Typescript rule for js file?

File: packages/ui/src/mocks/data/mockMembers.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { MemberFieldsFragment } from '../../api/queries'
+import rawMembers from './raw/members.json'
+
+export type MockVideo = MemberFieldsFragment
```
Comment (krzysztof-jelski) [inline packages/ui/src/mocks/data/mockMembers.ts:4]: This name (`MockVideo`) does not seem right here. `MockMember`?

**Review by krzysztof-jelski — APPROVED** [review]

File: packages/ui/src/mocks/data/mockMembers.ts (hunk 2 above)
Comment (jodator) [reply packages/ui/src/mocks/data/mockMembers.ts:4]: Yep, my bad :+1:

File: packages/ui/webpack.config.js (hunk 1 above)
Comment (jodator) [reply packages/ui/webpack.config.js]: It looks that just my IDE complains about it.

**Pull request merged**